src/main/java/org/logstash/netty/SslContextBuilder.java

```diff
@@ -129,58 +162,34 @@ public static String[] getDefaultCiphers(){
         }
     }
 
-    public static boolean isUnlimitedJCEAvailable(){
+    public static boolean isUnlimitedJCEAvailable() {
         try {
             return (Cipher.getMaxAllowedKeyLength("AES") > 128);
         } catch (NoSuchAlgorithmException e) {
             logger.warn("AES not available", e);
             return false;
         }
     }
-    public SslContextBuilder setCertificateAuthorities(String[] cert) {
-        certificateAuthorities = cert;
-        return this;
-    }
-
-    public SslContextBuilder setVerifyMode(SslClientVerifyMode mode) {
-        verifyMode = mode;
-        return this;
-    }
-
-    public File getSslKeyFile() {
-        return sslKeyFile;
-    }
-
-    public File getSslCertificateFile() {
-        return sslCertificateFile;
-    }
 
     public SslContext buildContext() throws Exception {
-        io.netty.handler.ssl.SslContextBuilder builder = io.netty.handler.ssl.SslContextBuilder.forServer(sslCertificateFile, sslKeyFile, passPhrase);
-
         if (logger.isDebugEnabled()) {
-            logger.debug("Available ciphers: " + SUPPORTED_CIPHERS);
-            logger.debug("Ciphers:  " + Arrays.toString(ciphers));
+            logger.debug(String.format("Available ciphers: %s", SUPPORTED_CIPHERS));
+            logger.debug(String.format("Ciphers: %s", Arrays.toString(ciphers)));
```

Comment:
We can leverage the log4j templating and avoid the move to String format


```suggestion
            logger.debug("Available ciphers: {}", SUPPORTED_CIPHERS);
            logger.debug("Ciphers: {}", Arrays.toString(ciphers));
```